<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresentorWindow</class>
 <widget class="QMainWindow" name="PresentorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="layoutMain">
    <item>
     <widget class="QGroupBox" name="grpMain">
      <property name="title">
       <string>vSongBook for Desktop</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolBar" name="toolBar">
         <property name="windowTitle">
          <string>toolBar</string>
         </property>
         <addaction name="actionClose"/>
         <addaction name="actionBold"/>
         <addaction name="actionFont"/>
         <addaction name="actionBigger"/>
         <addaction name="actionUp"/>
         <addaction name="actionDown"/>
         <addaction name="actionSmaller"/>
         <addaction name="actionTheme"/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutTop">
         <item>
          <widget class="QLabel" name="lblSongbook">
           <property name="text">
            <string>Songbook</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Song Title</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblKey">
           <property name="text">
            <string>Key</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutMiddle">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblContent">
           <property name="text">
            <string>Song Content</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutBottom">
         <item>
          <widget class="QPushButton" name="btnTheme">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/Color.png</normaloff>:/images/Color.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBold">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>36</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>B</string>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFont">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>36</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>F</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBigger">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>36</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSmaller">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>36</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblAuthor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Author</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblVerse">
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string>Verse</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
           <property name="indent">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUp">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/images.qrc">
             <normaloff>:/images/images/up_white.png</normaloff>:/images/images/up_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDown">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/images.qrc">
             <normaloff>:/images/images/down_white.png</normaloff>:/images/images/down_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUp">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionBigger">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionSmaller">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/images.qrc"/>
 </resources>
 <connections/>
</ui>
